tsconfig.json

```diff
@@ -8,14 +8,9 @@
     "useUnknownInCatchVariables": false,
     "resolveJsonModule": true,
     "declaration": false,
-    "lib": ["DOM"]
+    "lib": ["DOM"],
+    "skipLibCheck": true
```

Comment:
Why this addition?

Reply:
woops, I didn't realize this went through lol